code/tests/utilities/helpers/test_DocumentProcessorHelper.py

```diff
@@ -61,6 +72,22 @@ def azure_search_iv_indexer_helper_mock():
         yield mock
 
 
+def test_process_use_advanced_image_processing_skips_processing(
```

Comment:
This test doesn't look quite right, I think it should be:

`azure_search_helper_mock.return_value.get_vector_store.return_value = vector_store_mock`

and

`vector_store_mock.add_documents.assert_not_called()`

Any way we can avoid negative assertions?

Reply:
> Any way we can avoid negative assertions?

Possibly. I've not put too much effort here to be honest as the next ticket is to go ahead and actually implement this. This test will get replaced as part of that work.